doc/getting_started/installation.md

```diff
@@ -78,6 +78,38 @@ Make sure Panel is installed in the same environment as JupyterLab/Jupyter Noteb
 If you plan to use Panel in a non-Jupyter notebook environment, such as Google Colab or VSCode, refer to the [relevant how-to section](../how_to/notebook/other_nb.md).
 :::
 
+## Updating Panel
+
+:::{important}
+
+Instead of updating, we recommend that you create a new virtual environment from scratch and reinstall Panel to minimize the risk of potential issues.
+
+:::
+
+We suggest updating the dependencies `param` and `bokeh` along with `panel` and `watchfiles` to reduce the likelihood of problems.
+
+:::::{tab-set}
+
+::::{tab-item} pip
+:sync: pip
+
+```bash
+pip install --upgrade panel watchfiles param bokeh
```

Comment:
In my experience you need to include param or bokeh. Otherwise you can run into all sorts of problems.

Reply:
This is surprising to me.

Reply:
No longer relevant